### PR TITLE
Add ability to specify primitive type to encode payload

### PR DESCRIPTION
**Pull request by Pinkdesu**
- Add opportunity to specify primitive type to encode payload
- Fix header style

### CODE REVIEW

**Review by osipov-mit — APPROVED** [review]

File: website/frontend/src/components/pages/Send/children/MessageForm/MessageForm.tsx [hunk 1]
```diff
@@ -33,12 +34,44 @@ export const MessageForm: VFC<Props> = ({ id, meta, types, replyErrorCode }) =>
...
     value: 0,
     payload: types ? getPreformattedText(types) : '',
+    payloadType: 'Bytes',
     destination: id,
     __root: null,
   });
 
   const isReply = !!replyErrorCode;
 
+  const isMeta = useMemo(() => meta && Object.keys(meta).length > 0, [meta]);
```
Comment (nikitayutanov) [inline website/frontend/src/components/pages/Send/children/MessageForm/MessageForm.tsx:44]: Probably we can just check `types`

File: website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import styles from './SwitchableField.module.scss';
+
+type Props = {
+  name: string;
+  label: string;
+  defaultValue: string;
+};
+
+const SwitchableField = (props: Props) => {
```
Comment (nikitayutanov) [inline website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx:13]:
```suggestion
const PayloadType = (props: Props) => {
```
I don't think that it's reusable component, so specific name is fine

File: website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  const toggleSwitch = (event: ChangeEvent<HTMLInputElement>) => {
+    const checked = event.target.checked;
+
+    if (!checked) {
+      helpers.setValue(defaultValue);
+    }
+
+    setIsChecked(checked);
+  };
```
Comment (nikitayutanov) [inline website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx:27]:
```suggestion
useEffect(() => {
  if (!isChecked) {
    helpers.setValue(initialValue);
  }
}, [isChecked])

  const toggle = () => {
    setIsChecked(prevValue => !prevValue);
  };
```
imo it's cleaner this way

File: website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/index.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export * from "./SwitchableField"
```
Comment (nikitayutanov) [inline website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/index.ts:1]: fix trailing new line please

File: website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx (hunk 2 above)
Comment (Pinkdesu) [reply website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx:13]: It won't make it any worse.

File: website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx (hunk 3 above)
Comment (Pinkdesu) [reply website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx:27]: In my opinion it looks worse

File: website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import { useField } from 'formik';
+import { Input, Checkbox } from '@gear-js/ui';
+
+import styles from './SwitchableField.module.scss';
+
+type Props = {
+  name: string;
+  label: string;
+  defaultValue: string;
+};
```
[the remark this answers is not in the document]
Comment (Pinkdesu) [reply website/frontend/src/components/pages/Send/children/MessageForm/children/SwitchableField/SwitchableField.tsx]: It won't make it any worse.

**Review by nikitayutanov — APPROVED** [review]

**Pull request merged**